app/src/pages/SingleContent.vue

```diff
@@ -114,7 +144,7 @@ onMounted(async () => {
 
         <div
             class="mt-6 border-t border-zinc-200 pt-6 dark:border-zinc-500"
-            v-if="content.tags.length > 0"
+            v-if="content.tags.length != 0"
```

Comment:
Why did you change this from `>` to `!=`? It seems clearer to me if it's `>` - I would never expect it to be less than zero.

Reply:
@laurensgroeneveld I change like this because, in the test is not showing the tags when there is/are.

Reply:
@belljohan3 I think then we should investigate why that happens. Logically thinking:

- There are no tags: the length is 0
- There are tags: the length is greater than 0

So checking for if it's greater than 0 should always be sufficient, because the length can't be less than 0.

The only reason why I can imagine it's causing problems in the test is that `content.tags` itself is not an array but `undefined`. That means the `.length` will cause an error to occur. But then we should rather fix the test and not change the production code. 

Making the tests work is not a goal of itself: the tests are there to check that the code is working, so the code should generally not be changed to make the tests pass.